program/src/instruction.rs

```diff
@@ -1476,6 +1487,25 @@ pub fn create_mango_account(
     Ok(Instruction { program_id: *program_id, accounts, data })
 }
 
+pub fn set_delegate(
+    program_id: &Pubkey,
+    mango_group_pk: &Pubkey,
+    mango_account_pk: &Pubkey,
+    owner_pk: &Pubkey,
+    delegate_pk: &Pubkey,
+) -> Result<Instruction, ProgramError> {
+    let accounts = vec![
+        AccountMeta::new(*mango_group_pk, false),
```

Comment:
the comments suggest `mango_group_pk` is `readonly`

Reply:
thanks, correct, fixed